src/main/java/edu/cornell/kfs/fp/document/web/struts/RecurringDisbursementVoucherAction.java

```diff
@@ -0,0 +1,59 @@
+package edu.cornell.kfs.fp.document.web.struts;
+
+import java.util.ArrayList;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.struts.action.ActionForm;
+import org.apache.struts.action.ActionForward;
+import org.apache.struts.action.ActionMapping;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.batch.service.PaymentSourceExtractionService;
+import org.kuali.kfs.sys.context.SpringContext;
+
+import edu.cornell.kfs.fp.businessobject.RecurringDisbursementVoucherDetail;
+import edu.cornell.kfs.fp.document.RecurringDisbursementVoucherDocument;
+import edu.cornell.kfs.fp.service.RecurringDisbursementVoucherDocumentService;
+
+public class RecurringDisbursementVoucherAction extends CuDisbursementVoucherAction {
+	
+	protected static Log LOG = LogFactory.getLog(RecurringDisbursementVoucherAction.class);
+	
+	/**
```

Comment:
some of this indentation seems wonky; is it just github, or is the file not cleanly formatted? (tabs vs. spaces issue perhaps?)